packages/plugin-conversation/src/share-activity.js

```diff
@@ -316,7 +334,14 @@ ShareActivity.create = function create(conversation, object, spark) {
   });
 
   if (files) {
-    files.forEach((file) => share.add(file));
+    files.forEach((file) => {
+      if (isArray(file)) {
+        Reflect.apply(share.add, share, file);
+      }
+      else {
+        share.add(file);
+      }
+    });
```

Comment:
I may need to add a comment.  In this case file will be an array.  file[0] is the file, and file[1] are the options.